vochain/offchaindatahandler/offchaindatahandler.go

```diff
@@ -72,10 +72,11 @@ func (c *OffChainDataHandler) Commit(height uint32) error {
 		switch item.itemType {
 		case itemTypeExternalCensus:
 			log.Infof("importing external census %s", item.uri)
-			c.enqueueOffchainCensus(item.censusRoot, item.uri)
+			// AddToQueue() writes to a channel that might be full, so we don't want to block the main thread.
```

Comment:
i'd add here (or better, in func enqueueOffchainCensus comment)
https://github.com/vocdoni/vocdoni-node/blob/8abc3f60975fde6b2d34d89bf79b728a6b044dea/vochain/offchaindatahandler/census.go#L22

"safe for concurrent use, it just pushes an item to a channel"

since i needed to check that when i saw you prepending a "go" without further explanation

Reply:
same in enqueueMetadata
https://github.com/vocdoni/vocdoni-node/blob/b23bc28b0472f9a87801cdb1aaf8601e8b5db01b/vochain/offchaindatahandler/metadata.go#L9